i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-infor-ion-grid-restapi.md

```diff
@@ -0,0 +1,393 @@
+| CRITICALLOGWARNING    | Set critical thresholds on log-warning                                                                                                                              |                        |             |
+| WARNINGLOGERROR       | Set warning thresholds on log-error                                                                                                                                 |                        |             |
+| CRITICALLOGERROR      | Set critical thresholds on log-error                                                                                                                                |                        |             |
+| CRITICALSTATUS        | Set critical threshold for status. Can use special variables like: %{state}, %{name}, %{host\_name}, %{application\_name}, %{type}                                  | '%{state} !~ /online/' |             |
+| WARNINGSTATUS         | Set warning threshold for status (Default: "%{state} !~ /online/"). Can use special variables like: %{state}, %{name}, %{host\_name}, %{application\_name}, %{type} |                        |             |
+| WARNINGUPTIME         | Set warning thresholds on uptime                                                                                                                                    |                        |             |
+| CRITICALUPTIME        |Set critical thresholds on uptime                                                                                                                                    |                        |             |
+| EXTRAOPTIONS          | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                  |                        |             |
+
+</TabItem>
+</Tabs>
+
+3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
+
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
+
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
+
+```bash
+/usr/lib/centreon/plugins/centreon_infor_ion_grid_restapi.pl \
```

Comment:
@cg-tw : In original community PR there are real examples for both modes application and node, do you prefer them or we can let this one ?

`centreon_plugins.pl --plugin=apps::infor::ion::grid::api::plugin --mode=application --hostname='toto.domain' --port='23456' --proto='https' --api-username='blabla' --api-password='(/ç*/&"*' --filter-name='^M3BE_15.1$' --warning-status='%{online} =~ /true/ && %{state} =~ /^NOT OK$/i' --critical-status='%{online} =~ /false/ || %{state} =~ /^NOT RUNNING$/i'
OK: Application 'M3BE_15.1' [description: System Foundation for M3 BE] state is 'OK' [online: true] [started: true]`

`centreon_plugins.pl --plugin=apps::infor::ion::grid::api::plugin --mode=node --hostname='toto.domain' --port='23456' --proto='https' --api-username='blabla' --api-password='(/ç*/&"*' --filter-name='^Registry$' --filter-application-name='^SYSTEM$' --filter-host-name='^THEHOST$' --warning-status='%{state} !~ /online/' --change-perfdata='.*#(.*),$1'
OK: Node 'Registry' with PID '173' from application 'SYSTEM' on host 'THEHOST' state is 'online', Log error: 0, Log warning: 0, Uptime: 4M 2w 2d 2h 40m, CPU usage: 0.00%, Heap usage: 46.10MB/254.00MB (18.15%) | 'node.log.error.count'=0;;;0; 'node.log.warning.count'=0;;;0; 'node.cpu.usage.percentage'=0.00%;;;0;100 'node.heap.usage.percentage'=18.15%;0:90;;0;100`

Reply:
you can use the example from the community PR if we are sure all the data is anonymized